.storybook/main.js

```diff
@@ -8,30 +9,12 @@ module.exports = {
     // You can change the configuration based on that.
     // 'PRODUCTION' is used when building the static version of storybook.
 
-    config.module.rules.map((rule) => {
-      if (rule.oneOf) {
-        rule.oneOf = rule.oneOf.slice().map((subRule) => {
-          if (subRule.test instanceof RegExp && subRule.test.test('.scss')) {
-            return {
-              ...subRule,
-              use: [
-                ...subRule.use,
-                {
-                  loader: require.resolve('sass-resources-loader'),
-                  options: {
-                    resources: [
-                      path.resolve(__dirname, '../src/styles/index.scss')
-                    ]
-                  }
-                }
-              ],
-            }
-          }
-          return subRule;
-        });
-      }
-      return rule;
-    });
```

Comment:
Any idea if this was needed for anything? Didn't seem to work.

Reply:
Maybe that was for the pihla door project when I tried to make it work. It didn't in the end, but it still uses the index.css file from GDS. 

Anyways, good that you removed this.